bytecomp/translclass.ml

```diff
@@ -134,7 +131,8 @@ let rec build_object_init cl_table obj params inh_init obj_init cl =
                    [Lvar envs],
                    Location.none)]
       in
-      ((envs, (obj_init, normalize_cl_path cl path)
+      let path_lam = transl_class_path cl.cl_loc cl.cl_env path in
+      ((envs, (obj_init, path, path_lam)
         ::inh_init),
```

Comment:
Very minor: strange indentation for `::inh_init` (the cons applies to the triplet, not to `envs`), they could all be put on the same line.